### PR TITLE
More Ubuntu and debian env. dependencies

### PR DESCRIPTION
**Pull request by tfauck**
Signed-off-by: Thierry <tfauck@free.fr>

	modified:   cpu/ebizzy.py
	modified:   generic/nstress.py
	modified:   generic/service_check.py
	modified:   generic/service_check.py.data/services.cfg

### CODE REVIEW

File: generic/service_check.py [hunk 1]
```diff
@@ -42,14 +42,16 @@ def test(self):
             deps.extend(['libvirt-daemon'])
         if detected_distro.name == 'SuSE':
             deps.extend(['ppc64-diag', 'libvirt-daemon'])
-            if detected_distro.version >= 15:
+            if int(detected_distro.version) >= 15:
```
Comment (harish-24) [inline generic/service_check.py:45]: I presume `detected_distro.version` must already be an int. why is this change required?

Comment (PraveenPenguin) [reply generic/service_check.py:45]: ah i miss that @tfauck any update on this ?

**Comment by harish-24** [comment]
Closing due to inactivity, please re-open if needed.
